freqtrade/exchange/__init__.py

```diff
@@ -329,6 +341,53 @@ def get_ticker(self, pair: str, refresh: Optional[bool] = True) -> dict:
             logger.info("returning cached ticker-data for %s", pair)
             return self._cached_ticker[pair]
 
+    async def async_get_candles_history(self, pairs, tick_interval) -> List[Tuple[str, List]]:
+        # COMMENTED CODE IS FOR DISCUSSION: where should we close the loop on async ?
+        # loop = asyncio.new_event_loop()
+        # asyncio.set_event_loop(loop)
+        await self._api_async.load_markets()
+        input_coroutines = [self.async_get_candle_history(
+            symbol, tick_interval) for symbol in pairs]
+        tickers = await asyncio.gather(*input_coroutines, return_exceptions=True)
+        # await self._api_async.close()
+        return tickers
+
+    async def async_get_candle_history(self, pair: str, tick_interval: str,
+                                       since_ms: Optional[int] = None) -> Tuple[str, List]:
+        try:
+            # fetch ohlcv asynchronously
+            logger.debug("fetching %s ...", pair)
+
+            # Calculating ticker interval in second
+            interval_in_seconds = constants.TICKER_INTERVAL_MINUTES[tick_interval] * 60
```

Comment:
to answer - nope it doesn't as constants has a list of hour day week to minutes equiv

Reply:
i was asking myself the same earlier today - but no - `constants.TICKER_INTERVAL_MINUTES` converts the interval ("1d") to minutes (1440).

interval in seconds is then 1440 * 60

Reply:
CCXT provides exchange.parse_timeframe 
It may be useful to use CCXT's view of timeframes over our own constants 


https://github.com/ccxt/ccxt/blob/d76f474927f92d85734675ce8a86a1f754d4e63d/python/ccxt/base/exchange.py#L1235

Reply:
CCXT example to normalize sparse timestamps using provided functions. 

I don't think there is anything in it - other that aligning with CCXT and a possible help later if its changed to manage odd exchanges.  

https://github.com/ccxt/ccxt/blob/master/examples/py/normalize-sparse-candle-timestamps.py

Reply:
made it's own issue out of that - it's got nothing to do with async - but would be a great improvement of removing useless lines ...